tests/serverpod_test_server/test_e2e/auth_module_test.dart

```diff
@@ -172,4 +172,66 @@ void main() {
       expect(result, equals(true));
     });
   });
+
+  group('Given signed in user that wants to alter its user info', () {
+    setUp(() async {
+      var response = await client.authentication.authenticate(
+        'test@foo.bar',
+        'password',
+      );
+      assert(response.success, 'Failed to authenticate user');
+      await client.authenticationKeyManager
+          ?.put('${response.keyId}:${response.key}');
+      assert(
+          await client.modules.auth.status.isSignedIn(), 'Failed to sign in');
+    });
+
+    tearDown(() async {
+      await client.authenticationKeyManager?.remove();
+      await client.authentication.removeAllUsers();
+      await client.authentication.signOut();
+      assert(
+        await client.modules.auth.status.isSignedIn() == false,
+        'Still signed in after teardown',
+      );
+    });
+    test('when changing its user name then user name is updated.', () async {
+      final currentUserInfo = await client.modules.auth.status.getUserInfo();
+      expect(currentUserInfo, isNotNull,
+          reason: 'We expect to get UserInfo back from the auth module');
+      final String updatedFullName = '${currentUserInfo?.fullName}updated';
```

Comment:
Suggestion: Base suffix on the `userName`.

This test is for modifying the userName but on line we are adding a suffix for the `fullName`.

Reply:
Sloppy copy paste error. Will fix.